src/components/Modals/Onboarding/Onboarding.scss

```diff
@@ -8,7 +8,7 @@ input {
   background-color: none;
 }
 .onboarding {
-  padding: 1rem;
+  padding: 2rem 1rem;
```

Comment:
I think there are a lot of Profile + Onboarding css changes that we'll have due to the UI revamp next sem - may need to put changes in those files on hold for now